README.md

```diff
@@ -360,6 +360,14 @@ If set to 0: the player will show captions.
 
 If set to 1: the player won't show captions.
 
+##### `start`
+This parameter causes the player to begin playing the video at the given number of seconds from the start of the video. The parameter value is a positive integer.
+
+It works similarly to the `seekTo` method, and the `startSeconds` argument of `loadVideo` and `cueVideo`.
```

Comment:
What happens if you define both `start` and `startSeconds`? 

Why would one prefer using one over the other?

Reply:
IFramePlayerOptions `start` is useful in conjunction with a playlist, where `loadVideo`/`cueVideo` will override the playlist behavior and the player treats it as a single video instead: The playlist navigation button disappears.

To prevent that, you can pass the video ID during initialization, and the playlist using IFramePlayerOptions, and entirely skip calling `loadVideo`/`cueVideo`.

You could achieve the same using `seekTo` but that requires an extra JS call.

Reply:
Ok, sounds good!